cdap-common/src/main/java/co/cask/cdap/common/discovery/ResolvingDiscoverable.java

```diff
@@ -59,4 +59,9 @@ private InetSocketAddress resolve(InetSocketAddress bindAddress) {
     }
     return bindAddress;
   }
+
+  @Override
+  public String toString() {
+    return "ResolvingDiscoverable(" + String.valueOf(discoverable) + ")";
```

Comment:
It's no longer needed since starting in Twill 0.8.0, the `Discoverable` is a class and has the `toString` method implemented.